rb/lib/selenium/devtools/support/cdp_client_generator.rb

```diff
@@ -26,8 +26,9 @@ module DevTools
     module Support
       class CDPClientGenerator
         # Input JSON files are generated from PDL tasks.
-        DOMAIN_TEMPLATE_PATH = File.expand_path('cdp/domain.rb.erb', __dir__)
-        LOADER_TEMPLATE_PATH = File.expand_path('cdp/loader.rb.erb', __dir__)
+        DIR = __dir__ || ''
```

Comment:
Why is this necessary?

Reply:
When using __dir__ directly it's possible to return nil, and that produces and error type since the expectation for the file library is a string or an object that can receive _ToPath

<img width="837" alt="Screenshot 2024-03-13 at 22 26 20" src="https://github.com/SeleniumHQ/selenium/assets/33221555/d7ffd688-21e9-468a-9aa5-248de1698b64">
<img width="1355" alt="Screenshot 2024-03-13 at 22 25 56" src="https://github.com/SeleniumHQ/selenium/assets/33221555/8e15eaa9-5b09-47b1-96c5-f130bd71c9a7">

Reply:
That's unfortunate, though I would never expect __dir__ to return nil when called from a script.